source/release-notes/2.6.0.rst

```diff
@@ -25,23 +25,26 @@ Improvements
 ================================================================================
 
 * :jirabug:`PMM-1962`: The PMM Server API (via /v1/readyz) now also returns Grafana status information in addition to that for Prometheus.
-* :jirabug:`PMM-5854`: We've cleaned up the Service Details dashboards and removed some unused selectors.
-* :jirabug:`PMM-5775`: We've made it clearer which nodes are Primary and which are Secondary on MongoDB Instance dashboards.
+* :jirabug:`PMM-5854`: The Service Details dashboards were cleaned up and some unused selectors were removed.
+* :jirabug:`PMM-5775`: It is now clearer which nodes are Primary and which are Secondary on MongoDB Instance dashboards.
 * :jirabug:`PMM-5549`: We've updated PMM's Grafana component to the latest, v6.7.3.
-* :jirabug:`PMM-5393`: We've added a 'Node Summary' row to all services Summary and Details dashboards summarizing the system update, load average, RAM and memory.
-* :jirabug:`PMM-4778`: We've updated PMM's mongodb_exporter to the latest version, 0.11.0.
-* :jirabug:`PMM-5734`: We've added temporary files activity and utilization charts (rate & irate) to the PostgreSQL Instance overview.
-* :jirabug:`PMM-5695`: We've clarified the wording of the error message seen when using the --socket option incorrectly.
+* :jirabug:`PMM-5393`: There's a new 'Node Summary' row in the services Summary and Details dashboards summarizing the system update, load average, RAM and memory.
+* :jirabug:`PMM-4778`: The mongodb_exporter is now the latest version, v0.11.0.
+* :jirabug:`PMM-5734`: Temporary files activity and utilization charts (rate & irate) were added to the PostgreSQL Instance overview.
+* :jirabug:`PMM-5695`: The error message explains better when using the --socket option incorrectly.
 
 
 
 Bugs Fixed
 ================================================================================
 
-* :jirabug:`PMM-4829`: The MongoDB Exporter wasn't able to collect metrics from hidden nodes without either the latest driver or using the 'connect=direct' parameter.
+* :jirabug:`PMM-4829`: The MongoDB Exporter wasn't able to collect metrics from hidden nodes without either the latest driver or using the 'connect-direct' parameter.
 * :jirabug:`PMM-5056`: The average values for Query time in the Details and Profile sections were different.
-* :jirabug:`PMM-2717`: Updating the MongoDB Exporter resolves an error (``Failed to execute find query on 'config.locks': not found.``) when used with shardedCluster 3.6.4.
+* :jirabug:`PMM-2717`: Updating MongoDB Exporter resolves an error (``Failed to execute find query on 'config.locks': not found.``) when used with shardedCluster 3.6.4.
 * :jirabug:`PMM-4541`: MongoDB exporter metrics collection was including system collections from collStats and indexStats, causing "log bloat".
+* :jirabug:`PMM-5913`: Cluster=MongoDB filter returns only Total in QAN
+* :jirabug:`PMM-5909`: MySQL Overview dashboard shows No Data when user accessed it for second time
```

Comment:
This ticket is not fixed at the moment and could be postponed to the next release - we'll decide it in a few hours and then it may a need to remove this note from RN appear

Reply:
5909 removed.